File: is28/pavlov28/Python/lab2/prog2.py
import math
x1 = float(input("x1 = "))
y1 = float(input("y1 = "))
x2 = float(input("x2 = "))
y2 = float(input("y2 = "))
s = (x1-x2)**2+(y1-y2)**2
s = math.sqrt(s)
s = round(s, 3)
print(s)

File: is28/pavlov28/C++/lab2/prog2.cpp
#include <string>
#include <iostream>
#include <math.h>
#include <cmath>

using namespace std;

int main()
{
	float x1, x2, y1, y2, s;
//Нyжно найти расстояние междy точками
	cout << "x1: ";
	cin >> x1;
	cout << "y1: ";
	cin >> y1;
	cout << "x2: ";
	cin >> x2;
	cout << "y2: ";
	cin >> y2;
	s = ((x1-x2)*(x1-x2))+((y1-y2)*(y1-y2));
	s = sqrt(s);
	cout << "Расстояние междy точками равно " << round(s*1000)/1000 << endl;
	return 0;
}



